public/index.html

```diff
@@ -41,7 +41,7 @@
     <div class="app container">
       <div class="testNet">
         <div class="jumbotron">
-          <h1>RChain Annual Meeting 2020 Ballot <sup>(Beta Test)</sup></h1>
+          <h1>RChain Annual Meeting 2020 Ballot <sup>(Beta Test2)</sup></h1>
```

Comment:
Is this change on purpose?

Reply:
steve did that to see a change on the page. it is not needed.